silk/node/fifteen_four_dev_board.py

```diff
@@ -164,7 +167,7 @@ def get_device(self, name=None, sw_version=None):
         """
         Find an unused Needle, dev board, or other hardware.
         """
-        self.device = silk.hw.hw_resource.global_instance().get_hw_module(self._hwModel,
+        self.device = silk.hw.hw_resource.global_instance().get_hw_module(self.hwModel,
                                                                          name=name,
```

Comment:
nit: indent seems 1-space short

Reply:
Fixed the indent and also updated some doc strings. Can you please review again ?